Updating your SDK via git (#170)
gamemanual0/gm0: source/docs/software/using-android-studio.rst

To install any SDK tools such as platform tools or build tools, open the SDK manager and go to SDK Tools. Select the tools you want to install and install them the same way you would for the SDK platforms.

Updating the SDK using Git
--------------------------

> The SDK can be updated through the use of git and version control. In order to update the SDK, we will first need to set the source of the updates through git upstream, and then merge the updates into our current project.

Reviewer: ```suggestion
The SDK can be updated through the using the version control software Git. In order to update the SDK, we will first need to set the source of the updates using Git remotes, and then merge the updates into our current project.
```

> .. note:: Your FTC project needs to be set up as a Github Repository prior to following this guide.

Reviewer: ```suggestion
.. note:: Your FTC project needs to be set up as a Git repository prior to following this guide.
```

> Setting git upstream
> ^^^^^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
Setting Git Remote
^^^^^^^^^^^^^^^^^^
```

> #. Open up your project in android studio

Reviewer: ```suggestion
#. Open up your project in Android Studio
```

#. Open up the "terminal" tab
#. Paste in ``git remote add upstream https://github.com/FIRST-Tech-Challenge/FtcRobotController/`` and hit enter

> .. note:: The command above assumes you're using the FTC SDK. If you are using a modified version of the SDK as your base (e.g. Roadrunner Quickstart), replace the git link with one from the modified SDK repo

Reviewer: ```suggestion
.. note:: The command above assumes you're using the FTC SDK. If you are using a modified version of the SDK as your base (e.g. Road Runner Quickstart), replace the git link with one from the modified SDK repo
```

Updating the SDK
^^^^^^^^^^^^^^^^

In order to update the SDK, run ``git pull upstream master``

Merge Conflicts
^^^^^^^^^^^^^^^

> Merge conflicts occur when competing changes are made to the same line of a file. If you modified gradle files within your project, it is very likely you will run into a merge conflict when updating the sdk.

Reviewer: ```suggestion
Merge conflicts occur when competing changes are made to the same line of a file. If you modified Gradle files within your project, it is very likely you will run into a merge conflict when updating the SDK.
```
